experiments/inference.py

```diff
@@ -44,6 +46,48 @@ def evaluate(
     return trues, preds, losses
 
 
+def predict_image(
+    classifier: nn.Module,
+    classes: list[str],
+    device: str,
+    image
+) -> int:
+    """
+    Predict the class of image. Returns class index with top confidence.
+    """
+
+    classifier.to(device)
```

Comment:
Czy na pewno chcemy robić predykcję na GPU? Dobrze byłoby sprawdzić czy dla pojedynczego obrazu nie jest szybciej zrobić to na CPU - transfer danych do GPU trochę zajmuje, a nie robimy tutaj żadnych obliczeń modyfikujących model więc powinno być szybko.

Reply:
Trochę .... nanosekund to zajmuje 😄 
![image](https://user-images.githubusercontent.com/31924910/211798509-7f3ca21b-7494-463c-b258-dbc7c16b3ae0.png)

Spoczko - zmienione na CPU 👍